print/browserless/Dockerfile

```diff
@@ -1,6 +0,0 @@
-FROM browserless/chrome:1.50.0-puppeteer-12.0.1@sha256:ca9c1a56ff88ba91c3aac908713cfe92f4e308c941a64e408d00d26bf1de036a
-
-USER 0
-RUN apt-get update -y && apt-get install -y fonts-open-sans && rm -rf /var/lib/apt/lists/*
```

Comment:
we dont need fonts-open-sans anymore?

Reply:
So far we're loading fonts via Google CDN and this seems to work pretty well:
https://github.com/ecamp/ecamp3/blob/devel/print/assets/fonts.css

Reply:
True, the "eCamp3" text and page numbering on every page is in Open Sans, proving it works. The fonts can be most easily distinguished by the double/triple story lowercase g, and by the shape of the uppercase C.
![Screenshot from 2022-05-05 12-02-50](https://user-images.githubusercontent.com/7566995/166902210-374b7785-28fd-41e5-83c1-8913573f3309.png)

I must have gotten confused previously by Vuetify always overriding the body font to Roboto, which should be fixed in #2653.